meeting_minutes/2025/2025-07-30.md

```diff
@@ -0,0 +1,142 @@
+
+### Observers present
+
+Note: Observers of this committee that are ready to become Members should follow the specific instructions displayed the OASIS Open Notices tab.
+
+### Quorum
+
+- Quorum was reached (11 voting members present).
+
+## Agenda
+
+## Identification of TC voting members (Secretary)
+
+### Prospective voting members attending their first meeting
+
+### Members attaining voting rights at the end of this meeting
+
+- Sergii Demianchuk
+
+### Members losing voting rights if they have not joined this meeting by the time it ends
+
+- Christian Banse
+- JD Stefaniak
+
+### Members who previously lost voting rights who are attending this meeting
+
+- Feng Cao
+- Tobi Limmer
+
+### Members who have declared a leave of absence
+
+## Meeting Notes
+
+### GitHub Issues & Pull Requests
+
+#### CSAF Documentation GitHub Repository
+
+- PR [#88](https://github.com/oasis-open/csaf-documentation/pull/88)
+  - Motion set and passed already, but PR is still in Draft form pending Thomas Schmidt review. Thomas Schmidt will merge the PR.
+
+- PR [#90](https://github.com/oasis-open/csaf-documentation/pull/90)
+  - Thomas Schmidt sets the motion to approve the pull request to post the information and CFP for 2025 CSAF Community Days.
+  - Stefan seconds. No discussion, no objections. Motion passed.
+
+#### OASIS TC CSAF Repository
+
+- PR [#1108](https://github.com/oasis-tcs/csaf/pull/1108)
+  - Thomas Schmidt presents PR [#1108](https://github.com/oasis-tcs/csaf/pull/1108) which will serve as the feedback loop/comment resolution to bring back into our master branch.
+    - Contains the comments we received during the public review for CSD-01.
+    - Good amount of non-material changes and formatting adjustments.
+  - Nexts steps are to implement the open issues from the public review.
+    - A few topics for TC discussion, and a few that are ready for motions.
+  - The next editor revision should be prepared by our August meeting.
+  - As there are material questions, we will go to a draft again for another round of public review
+  - Outstanding issues include CVSS and SSVC. Working with those communites to have a good solution.
+- PR [#1086](https://github.com/oasis-tcs/csaf/issues/1086)
+  - Christoph Plutte presents the issue suggesting to add a `qualitative_severity_rating` to the metric element, while still emphasizing CVSS score is preferred option.
+  - Thomas suggests that if we are to have a `qualitative_severity_rating`, it could be good to have fixed categories with guidance on what category should be used.
+  - Christoph agrees that it is a good idea to have fixed values.
+  - Feng mentions the Provider Urgency field in CVSS v4.0 where the vendor can provide their opinion on if something is very important or not important.
+  - Christoph will work with the editors, Thomas and Stefan, to work on an implementation of the `qualitative_severity_rating` and draft guidance.
+
```

Comment:
@justmurphy As I can't remember and just for clarification: Did we had a motion on that?

Reply:
I think it is an action on Christoph and that fully suffices.